contrib/PKGBUILD

```diff
@@ -2,27 +2,27 @@ pkgname=alfis
 pkgver=0.3.12
```

Comment:
Don't you want to get version from last git tag?

Reply:
```
PKGVERSION=$(sh contrib/semver/version.sh --bare)
```

Reply:
Done in #23.